wikichange/assets/content.css

```diff
@@ -15,6 +15,20 @@ button:hover {
     color: white;
 }
 
+/* 
+these hover effect classes should be similar 
+to those of the actuall button above 
+*/
+.buttonNoHoverEffect {
```

Comment:
Can you rename to scaleButtonNoHoverEffect? to no confuse with the other buttons

Reply:
Fixed here [5a4c0c2](https://github.com/sukritsangvong/wp-browser-extension/pull/47/commits/5a4c0c2c011d375abb6a58ef7492736a8234d668)